src/devices/virtio-blk.c

```diff
@@ -0,0 +1,447 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+#include <assert.h>
+#include <fcntl.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "common.h"
```

Comment:
You don't have to include `common.h`. The top-level `Makefile` already specifies `CFLAGS += -include src/common.h -Isrc/`.